demo/web/src/main/notebooks/A. Real-time table ops/A2 Filter and decorate .md

```diff
@@ -0,0 +1,69 @@
+# Filter and decorate
+
+Now that we have some data, let's work with it. In this notebook, we show how to decorate and filter our data.
+
+Let's start by creating some daily data. This could simulate something like daily stock prices or daily temperatures.
+
+```python
+start_time = convertDateTime("2020-01-01T00:00:00 NY")
+
+time_offset = Period("1D")
+daily_data = create_random_table(365, start_time, time_offset)
+```
+
+Now we decorate the data by adding its day of the week.<!--TODO what is a good way to describe "decorate"?-->
+
+```python
+daily_data = daily_data.update("DayOfWeekInt = dayOfWeek(DateTime, TZ_NY)")
+```
+
+And now we convert the day of week to a string representation.
+
+```python
+import calendar
+
+def day_of_week_int_to_str(day_of_week):
+    return calendar.day_name[day_of_week-1]
+
+daily_data = daily_data.update("DayOfWeekStr = day_of_week_int_to_str(DayOfWeekInt)")
+```
+
+Let's do some filtering on this data. For this example, we filter based on values in our columns with simple boolean expressions.
```

Comment:
```suggestion
Deephaven provides a wealth of filtering methods for tables.  We will start by filtering using simple boolean expressions.
```

Reply:
we can link in a markdown notebook, right? link to "how to filter article"

Reply:
I believe so, I'm not sure about relative links so it may need to be an absolute link (https://deephaven.io/core/docs/how-to-guides/use-filters/)